aten/src/ATen/cpu/vec/sve/vec_common_sve.h

```diff
@@ -171,6 +202,19 @@ inline deinterleave2<float>(const Vectorized<float>& a, const Vectorized<float>&
                         Vectorized<float>(svuzp2_f32(a, b)));
 }
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ FLIP ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
These changes are not part of the overall goal of adding svl 128 and 512 support. In the future perhaps we should keep them in a different PR?

Reply:
Sure. Will do from next time.

Reply:
note that splitting these changes would help review go faster